File: LetSkole.Entities/Activity.cs
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations;
using System.Text;

namespace LetSkole.Entities
{

    public class Activity:EntityBase
    {
        public int UserId { get; set; }
        //public User User { get; set; }


        [Required]
        [StringLength(15)]
        public string Name { get; set; }
        [Required]
        [StringLength(256)]
        public string Description { get; set; }
        public DateTime StartDate { get; set; }
        public DateTime EndDate { get; set; }
        public bool Completed { get; set; }
        public DateTime DoDate { get; set; }
        public DateTime StartTime { get; set; }
        public DateTime EndTime { get; set; }
    }
}

File: frontend/letskole/src/models/activity.ts
export class Activity{
    id: number = 0;
    userId: number = 0;
    name: string = '';
    description: string = '';
    startDate: Date = new Date();
    endDate: Date = new Date(); 
    completed: boolean = false;
    startTime: Date = new Date();
    endTime: Date = new Date();
}